src/components/tests/dispenser-item.test.tsx

```diff
@@ -4,7 +4,7 @@ import * as RomiCore from '@osrf/romi-js-core-interfaces';
 import React from 'react';
 import fakeDispenserStates from '../../mock/data/dispenser-states';
 import DispenserItem from '../dispenser-item';
-import DispenserPanel from '../dispensers-panel';
+import renderer from 'react-test-renderer';
```

Comment:
Can you explain why we need to use `react-test-renderer` here and not in the other tests?